Engine/Components/DiscordEvents.hpp

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+void initDiscord(const char* id);
```

Comment:
Why did not you make this a controller rather than trying to introduce just a bunch of methods?

Reply:
This is my first time actually creating a new thing with c++ so I just used my knowledge of js to fill in anything I didn't know the proper implementation for